charts/incubator/cloudflared/questions.yaml

```diff
@@ -8,22 +8,23 @@ questions:
 # Include{replicas1}
 # Include{podSpec}
 # Include{containerMain}
+
+                                - variable: env
+                                  label: Image Environment
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                      - variable: token
+                                        label: Tunnel Token
+                                        description: Token for tunnel (Tunnel managed by Cloudflare not locally)
+                                        schema:
+                                          type: string
+                                          required: true
+                                          default: ""
+
```

Comment:
This is not an env. this is a custom key we have introduced in order to expose it on UI.
Leave it as it was before...